MVVM-C/Controllers/Main Controller/View/GoTCell.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  GoTCell.swift
+//  MVVM-C
+//
+//  Created by Usemobile on 09/06/20.
+//  Copyright © 2020 Jakub Homik. All rights reserved.
+//
+
+import UIKit
+
+class GoTCell: UITableViewCell {
+    
+    var viewModel: ArticleViewModel? {
+        didSet {
+            self.setupViewModel()
+        }
```

Comment:
what would happen if we delete didSet thing ? didSet is just checking if setupViewModel changed ? how this working under the hood ?

Reply:
The `didSet` is an state observer, that is called whenever the property value changes, so in this case is used to update the UI when the `viewModel` property is set. So when the `viewModel` value is changed, the `setupViewModel` method is called to update the UI according to that new value.